editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -823,6 +823,16 @@ void SpatialEditorViewport::_smouseexit() {
 	_remove_preview();
 }
 
+void SpatialEditorViewport::_sfocusentered() {
```

Comment:
you can use NOTIFICATION_FOCUS_ENTER and EXIT instead of connecting a signal

Reply:
Ok

Reply:
as per our conversation, signals must be used in this case